web/src/main/java/se/tre/freki/web/HttpServer.java

```diff
@@ -24,7 +24,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(HttpServer.class);
 
   /**
-   * Entry-point for the http server application. The assign program is normally not executed
+   * Entry-point for the http server application. The create program is normally not executed
```

Comment:
This is a typo. It should not have said assign to begin with. Can you change this to "web"?